<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPage</class>
 <widget class="QWidget" name="ConnectPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="titleLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iconLabel">
       <property name="pixmap">
        <pixmap resource="../../communi.qrc">:/communi-64.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;h1&gt;Communi&lt;/h1&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="sloganLabel">
       <property name="text">
        <string>&lt;small&gt;Communi is a simple and elegant cross-platform IRC client.&lt;/small&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="container">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <item row="0" column="1">
        <spacer name="topSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="leftSpacer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="connectionBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Connection</string>
         </property>
         <layout class="QFormLayout" name="connectionLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="displayNameLabel">
            <property name="text">
             <string>&amp;Name:</string>
            </property>
            <property name="buddy">
             <cstring>displayNameField</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="displayNameField">
            <property name="placeholderText">
             <string>Libera Chat</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hostLabel">
            <property name="text">
             <string>&amp;Host:</string>
            </property>
            <property name="buddy">
             <cstring>hostField</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="hostField">
            <property name="inputMethodHints">
             <set>Qt::ImhUrlCharactersOnly</set>
            </property>
            <property name="placeholderText">
             <string>irc.libera.chat</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="portLabel">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="buddy">
             <cstring>portField</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="portLayout">
            <item>
             <widget class="QSpinBox" name="portField">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>6697</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="secureBox">
              <property name="text">
               <string>&amp;Secure</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="expandButton">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextOnly</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="serverLabel">
            <property name="text">
             <string>&amp;Alternates:</string>
            </property>
            <property name="buddy">
             <cstring>serverField</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPlainTextEdit" name="serverField">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QWidget" name="rightSpacer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="identityBox">
         <property name="title">
          <string>Identity</string>
         </property>
         <layout class="QFormLayout" name="identityLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="nickNameLabel">
            <property name="text">
             <string>&amp;Nick names:</string>
            </property>
            <property name="buddy">
             <cstring>nickNameField</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="realNameLabel">
            <property name="text">
             <string>&amp;Real name:</string>
            </property>
            <property name="buddy">
             <cstring>realNameField</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="realNameField">
            <property name="placeholderText">
             <string>Communi %1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="userNameLabel">
            <property name="text">
             <string>&amp;User name:</string>
            </property>
            <property name="buddy">
             <cstring>userNameField</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="userNameField">
            <property name="placeholderText">
             <string>communi</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Pass&amp;word:</string>
            </property>
            <property name="buddy">
             <cstring>passwordField</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="passwordLayout">
            <item>
             <widget class="QLineEdit" name="passwordField">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="saslBox">
              <property name="text">
               <string>S&amp;ASL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QPlainTextEdit" name="nickNameField">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="bottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="copyrightLabel">
     <property name="text">
      <string>&lt;small&gt;Copyright (C) 2008-2016 The Communi Project&lt;/small&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>scrollArea</zorder>
  <zorder>buttonBox</zorder>
  <zorder>copyrightLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>displayNameField</tabstop>
  <tabstop>hostField</tabstop>
  <tabstop>portField</tabstop>
  <tabstop>secureBox</tabstop>
  <tabstop>expandButton</tabstop>
  <tabstop>serverField</tabstop>
  <tabstop>nickNameField</tabstop>
  <tabstop>realNameField</tabstop>
  <tabstop>userNameField</tabstop>
  <tabstop>passwordField</tabstop>
  <tabstop>saslBox</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../../communi.qrc"/>
 </resources>
 <connections/>
</ui>
